### PR TITLE
[zh] Localize 'infrastructure_as_a_service'

### LINKED ISSUE #866: [zh] Localize 'infrastructure_as_a_service'
- [x] Localize the content/en/infrastructure_as_a_service.md to content/zh/ in dev-zh.

### PR DESCRIPTION
**Pull request by Anddd7**
add `content/zh-cn/infrastructure_as_a_service.md`, fix #866

### CODE REVIEW

File: content/zh-cn/infrastructure_as_a_service.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+基础设施即服务，或IaaS，是一种[云计算](/cloud_computing/)服务模型，它提供[物理](/bare_metal_machine/)或[虚拟](/virtualization/)的计算、存储和网络资源，用户可按需按量进行结算支付。云服务提供商拥有和管理相关的软件和硬件设施，用户可以在公共、私有或混合云部署中使用这些软硬件资源。
+
+## 解决的问题
+
+在传统的本地设施部署时，组织往往受困于如何保证计算资源的高效使用。数据中心建立时必须考虑潜在的峰值需求，即使这样的需求只占其使用时间的1%。而在低需求时，这些计算资源又是空闲的。并且，如果工作负载超过了预期，则可能会出现计算资源的短缺。缺乏可伸缩性的本地设施会导致成本增加和资源使用效率低下。
+
+## 如何帮助
+
+通过IaaS，组织可以避免为应用程序购买和维护一整套计算和数据中心资源。按需使用的基础设施允许他们根据需要租用计算资源，并推迟大额的资金支出，或(CAPEX) (<https://en.wikipedia.org/wiki/Capital_expenditure>)，同时给予他们扩大或缩小的灵活性。
```
Comment (Rocksnake) [inline content/zh-cn/infrastructure_as_a_service.md:17]:
```suggestion
通过 IaaS，组织可以避免为应用程序购买和维护一整套计算和数据中心资源。按需使用的基础设施允许他们根据需要租用计算资源，并推迟大额的资金支出，或[CAPEX](https://en.wikipedia.org/wiki/Capital_expenditure)，同时给予他们扩大或缩小的灵活性。
```

File: content/zh-cn/infrastructure_as_a_service.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: 基础设施即服务 (IaaS)
+status: Completed
+category: 技术
+---
+
+## 是什么
+
+基础设施即服务，或IaaS，是一种[云计算](/cloud_computing/)服务模型，它提供[物理](/bare_metal_machine/)或[虚拟](/virtualization/)的计算、存储和网络资源，用户可按需按量进行结算支付。云服务提供商拥有和管理相关的软件和硬件设施，用户可以在公共、私有或混合云部署中使用这些软硬件资源。
```
Comment (Rocksnake) [inline content/zh-cn/infrastructure_as_a_service.md:9]:
```suggestion
基础设施即服务，或称 IaaS，是一种[云计算](/cloud_computing/)服务模型，它提供[物理](/bare_metal_machine/)或[虚拟](/virtualization/)的计算、存储和网络资源，用户可按需按量进行结算支付。云服务提供商拥有和管理相关的软件和硬件设施，用户可以在公共、私有或混合云部署中使用这些软硬件资源。
```

File: content/zh-cn/infrastructure_as_a_service.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## 解决的问题
+
+在传统的本地设施部署时，组织往往受困于如何保证计算资源的高效使用。数据中心建立时必须考虑潜在的峰值需求，即使这样的需求只占其使用时间的1%。而在低需求时，这些计算资源又是空闲的。并且，如果工作负载超过了预期，则可能会出现计算资源的短缺。缺乏可伸缩性的本地设施会导致成本增加和资源使用效率低下。
+
+## 如何帮助
+
+通过IaaS，组织可以避免为应用程序购买和维护一整套计算和数据中心资源。按需使用的基础设施允许他们根据需要租用计算资源，并推迟大额的资金支出，或(CAPEX) (<https://en.wikipedia.org/wiki/Capital_expenditure>)，同时给予他们扩大或缩小的灵活性。
+
+IaaS降低了试验或尝试新应用程序的前期成本，并提供了快速部署基础设施的工具。云提供商是开发或测试环境的绝佳选择，来帮助开发人员进行试验和创新。
```
Comment (Rocksnake) [inline content/zh-cn/infrastructure_as_a_service.md:19]:
```suggestion
IaaS 降低了试验或尝试新应用程序的前期成本，并提供了快速部署基础设施的工具。云提供商是开发或测试环境的绝佳选择，来帮助开发人员进行试验和创新。
```

**Comment by Jacob953** [comment]
Do not Pull Request directly to main, dev-zh first please! @Anddd7

**Comment by Jacob953** [comment]
Hi, @Anddd7!
How about joining #glossary-localizations & #glossary-localization-chinese? We can help you more easily in how-to-contribute. BTW, Team Chinese chats more on WeChat. ;)

**Comment by Anddd7** [comment]
> Hi, @Anddd7! How about joining #glossary-localizations & #glossary-localization-chinese? We can help you more easily in how-to-contribute. BTW, Team Chinese chats more on WeChat. ;)

Sure, already joined slack, can you send me the wechat qr in slack?

File: content/zh-cn/infrastructure_as_a_service.md (hunk 1 above)
Comment (Anddd7) [reply content/zh-cn/infrastructure_as_a_service.md]: resolved

File: content/zh-cn/infrastructure_as_a_service.md (hunk 3 above)
Comment (Anddd7) [reply content/zh-cn/infrastructure_as_a_service.md]: resolved

File: content/zh-cn/infrastructure_as_a_service.md (hunk 2 above)
Comment (Anddd7) [reply content/zh-cn/infrastructure_as_a_service.md]: resolved

**Comment by Anddd7** [comment]
> Do not Pull Request directly to main, dev-zh only please! @Anddd7

Already changed the base, 

> Do not Pull Request directly to main, dev-zh only please! @Anddd7

Got it, I will switch the branch and re-raise the PR

**Pull request closed without merging**